commit be08eda54c7cf833ccfa4b8d1f1b1d668c26af66
Author: Andreas Jaeger <aj@suse.de>
Date:   Wed May 23 09:27:39 2012 +0200

    Update from Linux 3.4

[ Partial, unrelated bits not included. ]

2012-05-23  Andreas Jaeger  <aj@suse.de>

	* sysdeps/unix/sysv/linux/powerpc/bits/mman.h
	(MADV_DONTDUMP,MADV_DODUMP): New macros from Linux 3.4.
	macro.
	* sysdeps/unix/sysv/linux/s390/bits/mman.h
	(MADV_DONTDUMP,MADV_DODUMP): Likewise.
	* sysdeps/unix/sysv/linux/i386/bits/mman.h
	(MADV_DONTDUMP,MADV_DODUMP): Likewise.
	* sysdeps/unix/sysv/linux/x86_64/bits/mman.h
	(MADV_DONTDUMP,MADV_DODUMP): Likewise.

diff -Nrup a/sysdeps/unix/sysv/linux/i386/bits/mman.h b/sysdeps/unix/sysv/linux/i386/bits/mman.h
--- a/sysdeps/unix/sysv/linux/i386/bits/mman.h	2010-05-04 05:27:23.000000000 -0600
+++ b/sysdeps/unix/sysv/linux/i386/bits/mman.h	2012-09-27 11:44:26.985150148 -0600
@@ -1,5 +1,5 @@
 /* Definitions for POSIX memory map interface.  Linux/i386 version.
-   Copyright (C) 1997,2000,2003,2005,2006,2009 Free Software Foundation, Inc.
+   Copyright (C) 1997-2012 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
@@ -94,6 +94,9 @@
 # define MADV_DOFORK	  11	/* Do inherit across fork.  */
 # define MADV_MERGEABLE	  12	/* KSM may merge identical pages.  */
 # define MADV_UNMERGEABLE 13	/* KSM may not merge identical pages.  */
+# define MADV_DONTDUMP	  16    /* Explicity exclude from the core dump,
+                                   overrides the coredump filter bits.  */
+# define MADV_DODUMP	  17	/* Clear the MADV_DONTDUMP flag.  */
 # define MADV_HWPOISON	  100	/* Poison a page for testing.  */
 #endif
 
diff -Nrup a/sysdeps/unix/sysv/linux/powerpc/bits/mman.h b/sysdeps/unix/sysv/linux/powerpc/bits/mman.h
--- a/sysdeps/unix/sysv/linux/powerpc/bits/mman.h	2010-05-04 05:27:23.000000000 -0600
+++ b/sysdeps/unix/sysv/linux/powerpc/bits/mman.h	2012-09-27 11:43:37.161351259 -0600
@@ -96,6 +96,9 @@
 # define MADV_DOFORK	  11	/* Do inherit across fork.  */
 # define MADV_MERGEABLE	  12	/* KSM may merge identical pages.  */
 # define MADV_UNMERGEABLE 13	/* KSM may not merge identical pages.  */
+# define MADV_DONTDUMP	  16    /* Explicity exclude from the core dump,
+                                   overrides the coredump filter bits.  */
+# define MADV_DODUMP	  17	/* Clear the MADV_DONTDUMP flag.  */
 # define MADV_HWPOISON	  100	/* Poison a page for testing.  */
 #endif
 
diff -Nrup a/sysdeps/unix/sysv/linux/s390/bits/mman.h b/sysdeps/unix/sysv/linux/s390/bits/mman.h
--- a/sysdeps/unix/sysv/linux/s390/bits/mman.h	2010-05-04 05:27:23.000000000 -0600
+++ b/sysdeps/unix/sysv/linux/s390/bits/mman.h	2012-09-27 11:44:49.745059646 -0600
@@ -94,6 +94,9 @@
 # define MADV_DOFORK	  11	/* Do inherit across fork.  */
 # define MADV_MERGEABLE	  12	/* KSM may merge identical pages.  */
 # define MADV_UNMERGEABLE 13	/* KSM may not merge identical pages.  */
+# define MADV_DONTDUMP	  16    /* Explicity exclude from the core dump,
+                                   overrides the coredump filter bits.  */
+# define MADV_DODUMP	  17	/* Clear the MADV_DONTDUMP flag.  */
 # define MADV_HWPOISON	  100	/* Poison a page for testing.  */
 #endif
 
diff -Nrup a/sysdeps/unix/sysv/linux/x86_64/bits/mman.h b/sysdeps/unix/sysv/linux/x86_64/bits/mman.h
--- a/sysdeps/unix/sysv/linux/x86_64/bits/mman.h	2010-05-04 05:27:23.000000000 -0600
+++ b/sysdeps/unix/sysv/linux/x86_64/bits/mman.h	2012-09-27 11:42:08.314725733 -0600
@@ -1,5 +1,5 @@
 /* Definitions for POSIX memory map interface.  Linux/x86_64 version.
-   Copyright (C) 2001, 2003, 2005, 2006, 2009 Free Software Foundation, Inc.
+   Copyright (C) 2001-2012 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
@@ -95,6 +95,9 @@
 # define MADV_DOFORK	  11	/* Do inherit across fork.  */
 # define MADV_MERGEABLE	  12	/* KSM may merge identical pages.  */
 # define MADV_UNMERGEABLE 13	/* KSM may not merge identical pages.  */
+# define MADV_DONTDUMP	  16    /* Explicity exclude from the core dump,
+                                   overrides the coredump filter bits.  */
+# define MADV_DODUMP	  17	/* Clear the MADV_DONTDUMP flag.  */
 # define MADV_HWPOISON	  100	/* Poison a page for testing.  */
 #endif
 
